RELEASES.md

```diff
@@ -77,18 +77,30 @@ Support horizons will be defined corresponding to a release branch, identified
 by `<major>.<minor>`. Releases branches will be in one of several states:
 
 - __*Next*__: The next planned release branch.
-- __*Active*__: The release branch is currently supported and accepting patches.
+- __*Active*__: The release is a stable branch which is currently supported and accepting patches.
 - __*Extended*__: The release branch is only accepting security patches.
+- __*LTS*__: The release is a long term stable branch which is currently supported and accepting patches.
 - __*End of Life*__: The release branch is no longer supported and no new patches will be accepted.
 
-Releases will be supported up to one year after a _minor_ release. This means that
+Releases will be supported at least one year after a _minor_ release. This means that
 we will accept bug reports and backports to release branches until the end of
 life date. If no new _minor_ release has been made, that release will be
 considered supported until 6 months after the next _minor_ is released or one year,
 whichever is longer. Additionally, releases may have an extended security support
 period after the end of the active period to accept security backports. This
 timeframe will be decided by maintainers before the end of the active status.
 
+Long term stable (_LTS_) releases will be supported for at least three years after
+their initial _minor_ release. These branches will accept bug reports and
+backports until the end of life date. They may also accept a wider range of
+patches than non-_LTS_ releases to support the longer term maintainability of the
+branch, including dependency and other version updates which are needed to
+ensure each release is built with fully supported dependencies and remains
+usable by containerd clients. There must be at least a 6 month overlap between
```

Comment:
Is there a set of feature/enhancement changes we'd backport as well?  For example, if CRI changes are made during the support window that interact with currently-supported Kubernetes versions?

Reply:
CRI changes are very likely since we will want the LTS to usable with Kubernetes releases. I think we evaluate the changes based on impact and client need. I could see backporting some feature changes for nerdctl and Docker as well.